app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,161 @@
+
+    <Button android:id="@+id/button_six"
+        style="@style/button_style"
+        android:text="@string/activity_main_calculator_six_text"
+        app:layout_constraintBottom_toTopOf="@id/button_three"
+        app:layout_constraintEnd_toEndOf="@id/button_three"
+        app:layout_constraintStart_toStartOf="@id/button_three" />
+
+    <Button android:id="@+id/button_multiply"
+        style="@style/button_style"
+        android:text="@string/activity_main_calculator_multiply_text"
+        app:layout_constraintBottom_toTopOf="@id/button_subtraction"
+        app:layout_constraintEnd_toEndOf="@id/button_subtraction"
+        app:layout_constraintStart_toStartOf="@id/button_subtraction" />
+
+    <Button android:id="@+id/button_one"
+        style="@style/button_style"
+        android:text="@string/activity_main_calculator_one_text"
+        app:layout_constraintBottom_toTopOf="@id/button_zero"
+        app:layout_constraintEnd_toEndOf="@id/button_zero"
+        app:layout_constraintStart_toStartOf="@id/button_zero" />
+
+    <Button android:id="@+id/button_two"
+        style="@style/button_style"
+        android:text="@string/activity_main_calculator_two_text"
+        app:layout_constraintBottom_toTopOf="@id/button_clean"
+        app:layout_constraintEnd_toEndOf="@id/button_clean"
+        app:layout_constraintStart_toStartOf="@id/button_clean" />
+
+    <Button android:id="@+id/button_three"
+        style="@style/button_style"
+        android:text="@string/activity_main_calculator_three_text"
+        app:layout_constraintBottom_toTopOf="@id/button_equals"
+        app:layout_constraintEnd_toEndOf="@id/button_equals"
+        app:layout_constraintStart_toStartOf="@id/button_equals" />
+
+    <Button android:id="@+id/button_subtraction"
+        style="@style/button_style"
+        android:text="@string/activity_main_calculator_subtraction_text"
+        app:layout_constraintBottom_toTopOf="@id/button_add"
+        app:layout_constraintEnd_toEndOf="@id/button_add"
+        app:layout_constraintStart_toStartOf="@id/button_add" />
+
+    <Button android:id="@+id/button_zero"
+        style="@style/button_style"
+        android:layout_marginBottom="@dimen/activity_main_marginBottom_button"
+        android:text="@string/activity_main_calculator_zero_text"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/button_clean"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintHorizontal_chainStyle="spread"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <Button android:id="@+id/button_clean"
+        style="@style/button_style"
+        android:layout_marginBottom="@dimen/activity_main_marginBottom_button"
+        android:text="@string/activity_main_calculator_clean_text"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/button_equals"
```

Comment:
please, remove the "+" , Its not necesary to identify an ID

Reply:
Solved!